starter_code/app.js

```diff
@@ -1,21 +1,56 @@
-
-const express = require('express');
-const hbs     = require('hbs');
-const app     = express();
-const path    = require('path');
-const PunkAPIWrapper = require('punkapi-javascript-wrapper');
+//setup
+const express = require("express");
+const hbs = require("hbs");
+const app = express();
+const path = require("path");
+const PunkAPIWrapper = require("punkapi-javascript-wrapper");
 const punkAPI = new PunkAPIWrapper();
 
-app.set('view engine', 'hbs');
-app.set('views', __dirname + '/views');
-app.use(express.static(path.join(__dirname, 'public')));
+app.set("view engine", "hbs");
+app.set("views", __dirname + "/views");
 
+hbs.registerPartials(path.join(__dirname, "/views/partials"));
 
+app.use(express.static(path.join(__dirname, "public")));
 
-app.get('/', (req, res, next) => {
-  res.render('index');
+app.get("/", (req, res, next) => {
+  res.render("index");
 });
 
+app.get("/beers", (req, res, next) => {
+  punkAPI
+    .getBeers()
+    .then(beers => {
+      console.log(beers);
+      res.render("beers", { beers });
+    })
+    .catch(error => {
+      console.log(
+        "THERE WAS AN ERROR when trying to catch the array of 25 beers",
```

Comment:
I really like how you've made the error message personal to the get request. This is going to be super helpful during project week